src/auth/OAUthWithIDTokenValidation.js

```diff
@@ -0,0 +1,92 @@
+var jwt = require('jsonwebtoken');
+var jwksClient = require('jwks-rsa');
+var Promise = require('bluebird');
+
+var ArgumentError = require('rest-facade').ArgumentError;
+
+/**
+ * @class
+ * Abstracts the `oauth.create` method with additional id_token validation\
```

Comment:
Is the "\" needed?

Reply:
it's good to have this because the jsdocs website will show as a `formatted` text